src/script/media/MediaDevicesHandler.ts

```diff
@@ -343,6 +359,34 @@ export class MediaDevicesHandler {
     return screenSources;
   }
 
+  /**
+   * Enumerates all known Media Devices Infos of a System.
+   * @param forceCamera If `forceCamera=true`, then an audio and video track is created to enforce the device access
+   * permissionm
+   * @returns All enumerated MediaDeviceInfos
+   */
+  private async enumerateMediaDevices(forceCamera: boolean = false): Promise<MediaDeviceInfo[]> {
```

Comment:
You don't have to assign type there, because default value assign type to arg ;)

```suggestion
  private async enumerateMediaDevices(forceCamera = false): Promise<MediaDeviceInfo[]> {
```